python/tk_multi_launchapp/base_launcher.py

```diff
@@ -87,6 +87,11 @@ def _register_launch_command(
         if command_name.endswith("..."):
             command_name = command_name[:-3]
 
+        if group and menu_name.startswith(group):
```

Comment:
Yes. I think it's not a good idea to group multiple products together though. But, if you wanted to do this, you'd create a group called **Nuke Products** or **Nuke Family**. That does make the default command not so useful as it depends on the order in the menu which means you'd pretty much have to click the arrow to find what you want. Anyway, we can discuss other options, but I think this is better for the common case where you're grouping by Product and you don't want the duplication in the submenu. This also prevents the duplication of the group name in the recents buttons. I can demo for you guys tomorrow and we can discuss.